.github/workflows/apple.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
       timeout: 90
       secrets-env: BUILD_CERTIFICATE_BASE64 EXECUTORCH_DEMO_BUILD_PROVISION_PROFILE_BASE64 KEYCHAIN_PASSWORD
-      upload-artifact: ios-apps
+      upload-artifact: ios-demo-app
```

Comment:
Changing the upload artifact name here is needed to address https://github.com/actions/upload-artifact/issues/480.  In v4, we couldn't upload multiple artifacts with the same name anymore.